pandas/tests/io/test_sql.py

```diff
@@ -1374,6 +1390,13 @@ def test_bigint(self):
 
         tm.assert_frame_equal(df, result)
 
+    def test_reading_ints(self):
+        # BigInt should be read as int64, Int should be read as int32, and SmallInt should be read as int16
+        df = sql.read_sql_query("SELECT * FROM types_test_data", self.conn)
```

Comment:
Is this already checked in an existing test? If so probably makes sense to update that test rather than creating a new one altogether

Reply:
@WillAyd I added the tests to a similar test, thanks for the suggestion!